test/02-generate.js

```diff
@@ -31,6 +33,29 @@ describe('Module - generate', function () {
         }, assertCritical(target, expected, done));
     });
 
+    it('should generate critical-path CSS from CSS files passed as Vinyl objects', function (done) {
```

Comment:
Should generate the same results as the first test, but with stylesheets passed as Vinyl objects instead of links.